dev-db/sqlitebrowser/sqlitebrowser-3.11.1-r1.ebuild

```diff
@@ -1,67 +0,0 @@
-# Copyright 1999-2020 Gentoo Authors
```

Comment:
Our preferred policy is to make one commit adding the new version, and then another removing old. Possibly include some delay between the two in case it turned out the new version is buggy and people'd like to downgrade.